src/cpu/dynarmic_wrapper/build.rs

```diff
@@ -72,16 +73,11 @@ fn main() {
             .join("build/externals/fmt")
             .join(build_type_windows()),
     );
-    // fmtd isn't on Linux
-    if !cfg!(target_os = "linux") {
-        link_lib(if cfg!(debug_assertions) {
-            "fmtd"
-        } else {
-            "fmt"
-        });
+    link_lib(if cfg!(debug_assertions) {
+        "fmtd"
```

Comment:
Ah, is it named fmtd on Linux too then, when statically linked?

Reply:
Seems like it. Compiles fine for me on debug and release with this.